src/components/select/editForm/Select.edit.data.js

```diff
@@ -144,22 +144,24 @@ export default [
       const newItems = [];
 
       eachComponent(form.components, (component, path) => {
-        newItems.push({
-          label: component.label || component.key,
-          key: path
-        });
+        if (component.input) {
+          newItems.push({
+            label: component.label || component.key,
+            key: `data.${path}`
+          });
+        }
       });
 
       return newItems;
     },
     data: {
-      url: '/form/{{ data.resource }}'
+      url: '/form/{{ data.data.resource }}'
```

Comment:
This does not seem right.... why do we have to reference the data within data here?